recipes/openfaas.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: kbrew
+app:
+  args:
+    functionNamespace: openfaas-fn
+    generateBasicAuth: true
```

Comment:
What other elements are important from values.yaml? imagePullPolicy for instance

Reply:
I took the defaults from https://artifacthub.io/packages/helm/openfaas/openfaas.

Any suggestion?

Reply:
I suggested one above the Helm chart README is the best place to look - https://github.com/openfaas/faas-netes/blob/master/chart/openfaas/README.md - or values.yaml

Reply:
most of them have already default, reasonable values. I always start openfaas like this for lokal dev. And it works perfectly.